src_new/config.js

```diff
@@ -318,7 +323,7 @@ exports.getTestGroupDetails = function () {
 // This will keep toObject config as is and only merge objects common in both from and toobject 
 exports.getMergedConfig = function(toObject, fromObject){
 	for(var key in fromObject){
-		if(!toObject[key]) {
+		if(!toObject.hasOwnProperty(key)) {
```

Comment:
![Codacy](https://codacy.pubmatic.com:443/assets/images/favicon.png) Issue found: [Do not access Object.prototype method 'hasOwnProperty' from target object.](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=15011)